camel-core/src/main/java/org/apache/camel/component/xslt/XsltEndpoint.java

```diff
@@ -71,6 +72,8 @@
     @UriParam
     private boolean saxon;
     @UriParam(label = "advanced")
+    private List<Object> saxonExtensionFunctions;
```

Comment:
Yeah you would need to use org.apache.camel.util.EndpointHelper#resolveReferenceListParameter in the component. You can git grep to find other components using it. Then the type in the endpoint can be the same.